code/subscription.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kreis4 Flower Shop - Order Flowers Online</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+  <header> 
+    <nav class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </nav>
+
+    <menu>
+        <a href="./index.html">Home</a>
+        <a href="./our-flowers.html">Flowers</a>
+        <a href="./subscription.html">Subscription</a>
+        <a href="./faqs.html">FAQs</a>
+        <a href="./about.html">About</a>
+    </menu>
+
+    <div class="heading-desktop">
+      <h1>Welcome to Kreis4 Flowers</h1>
+      <h2>Fresh Flowers All Year Round</h2>
+  </div>
+  <div class="heading-mobile">
+      <h1>Kreis4 Flowers</h1>
+  </div>
+</header>
+
+<main>
+  <h1>Subscription</h1>
+  <h2>Subscribe here to our flower delivery service</h2>
+</main>
+
+<form action="http://httpbin.org/anything" method="POST">
+  <h3>Contact Details</h2>
+  <div class="formParent">
+    <div class="formBlock">
+      <label>
+        <span>First name:</span>
+        <input type="text" name="firstName" required minlength="2" placeholder="Tina">
```

Comment:
When we have several attributes it's nice to split them up for better readability:
```
<input
  required
  type="text"
  name="firstName"
  minlength="2"
  placeholder="Tina">
```